wercker.yml

```diff
@@ -74,16 +74,22 @@ test:
   - script:
       name: Setup Test environment
       code: |
+        export version=1.0.8 # latest stable version
+        export arch=amd64
```

Comment:
Nitpick; don't think you need to export these, they _should_ work fine as just variables :)